src/libstore/build/local-derivation-goal.cc

```diff
@@ -195,7 +195,7 @@ void LocalDerivationGoal::tryLocalBuild()
         else if (settings.sandboxMode == smDisabled)
             useChroot = false;
         else if (settings.sandboxMode == smRelaxed)
-            useChroot = derivationType.isSandboxed() && !noChroot;
+            useChroot = derivationType->isSandboxed() && !noChroot;
```

Comment:
Better to use `derivationType().value()` here, since it's not obvious whether `derivationType` is initialized. (An assertion failure would be better than an exception though...)

Reply:
I added one a few lines above.